src/modules/olMap/handler/routing/OlRoutingHandler.js

```diff
@@ -556,33 +561,27 @@ export class OlRoutingHandler extends OlLayerHandler {
 		}
 	}
 
-	async _requestRoute(defaultCategoryId, alternativeCategoryIds, coordinates3857) {
+	async _requestRouteAndDisplayRoute(defaultCategoryId, alternativeCategoryIds, coordinates3857) {
```

Comment:
I suggest `_requestAndDisplayRoute`, unless there is a good reason to separate the two operations (request, display) more decisively for the reader